cmake/InstallAndPackage.cmake

```diff
@@ -138,6 +138,13 @@ elseif(WIN32)
     endif()
 
     set(CPACK_PACKAGE_FILE_NAME "openttd-#CPACK_PACKAGE_VERSION#-windows-${CPACK_SYSTEM_NAME}")
+
+    if(CMAKE_SIGNING_CERTIFICATE)
```

Comment:
Here yet another variation of naming a variable :) I suggest we make them all similar .. here there is no mention of `WIN` (or `WINDOWS`), for example

Reply:
I will rename it. In this case it is the certificate CN that's needed.